### PR TITLE
#13 Test for fingerprint authentication activity after login

### PR DESCRIPTION
**Pull request by Landry333**
### Related Issues
Related to issue #13

### Description
Test class for the FingerprintAuthenticationActivity

### How to check PR
Run the test and check if it tests passed

Important files: FingerprintAuthenticationActivityTest.java

### CODE REVIEW

File: app/src/main/java/com/example/bigowlapp/activity/FingerprintAuthenticationActivity.java [hunk 1]
```diff
@@ -127,6 +128,9 @@ protected void onStart() {
         if (homePageViewModel == null) {
             homePageViewModel = new ViewModelProvider(this).get(HomePageViewModel.class);
         }
+        if (phoneNumberFormatter == null) {
+            phoneNumberFormatter = new PhoneNumberFormatter(this);;
```
Comment (refatK) [inline app/src/main/java/com/example/bigowlapp/activity/FingerprintAuthenticationActivity.java]: Remove extra semi-colon

**Review by refatK — COMMENTED** [review]
Just fix this small quality thing (also a sonar code smell), and you should be good

File: app/src/main/java/com/example/bigowlapp/activity/FingerprintAuthenticationActivity.java (hunk 1 above)
Comment (ShawnGuerra-Bautista) [inline app/src/main/java/com/example/bigowlapp/activity/FingerprintAuthenticationActivity.java]: Just like what @refatK stated.

This semi-colon should be removed; it's also a code smell according to sonarqube.

**Review by ShawnGuerra-Bautista — CHANGES_REQUESTED** [review]
The tests work, and it covers the majority of the cases.

There's only one small change, then it's pretty much good to go.

**Review by ShawnGuerra-Bautista — APPROVED** [review]

File: app/src/main/java/com/example/bigowlapp/activity/FingerprintAuthenticationActivity.java (hunk 1 above)
Comment (Landry333) [reply app/src/main/java/com/example/bigowlapp/activity/FingerprintAuthenticationActivity.java]: Ok. That has been done

**Review by refatK — APPROVED** [review]
Good tests and working and expected.

The coverage for `FingerprintAuthenticationActivity.java` went from 0% to 81.57%. Good job!

**Pull request merged**